spec/models/countries_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require_relative File.join(File.dirname(__FILE__), '..', '..', 'app', 'models', 'countries')
```

Comment:
I didn't realise spec_helper was already required.

Required the full rails environment is a ~3000% slowdown loading this file alone, but obviously makes no difference for a full run. I'll change it for consistency, but a better solution would be removing the require rails_helper lines from any specs that don't actually depend on rails.